File: server/tests/fixtures/sample.rs
// Simple Rust fixture for DAP integration testing
fn calculate_sum(x: i32, y: i32) -> i32 {
    let result = x + y;
    let doubled = result * 2;
    println!("Result: {}", doubled);
    doubled
}

fn main() {
    let a = 10;
    let b = 20;
    let sum = calculate_sum(a, b);
    println!("Sum: {}", sum);
}

File: tests/fixtures/sample.py
# Simple Python fixture for DAP integration testing
def calculate_sum(x, y):
    result = x + y
    doubled = result * 2
    print(f"Result: {doubled}")
    return doubled

def main():
    a = 10
    b = 20
    sum_value = calculate_sum(a, b)
    print(f"Sum: {sum_value}")

if __name__ == "__main__":
    main()
